src/calendar/CalendarMonth.jsx

```diff
@@ -32,6 +26,12 @@ const CalendarMonth = React.createClass({
     value: CustomPropTypes.momentOrMomentRange,
   },
 
+  componentWillMount() {
+    const lang = moment().localeData();
```

Comment:
In our case in my team, we have either `en` or `ar`, so from one to the other, there's a big change, and It requires the global moment object to set the locale, as the whole app requires different locale.. 

I don't mind having it prop driven, maybe it will allow more control on the component itself, but bear in mind that if I `require('moment/locale/${language}')`, also the internal state of the calendar will change.. So I believe that making it prop driven will require more work as:
- we have to isolate the moment object used in the component, initialize it with the prop passed, so it won't be affected by the global `require('moment/locale...')`

For us is fine as we just are working with all dates and hours with `moment`, so when the lang changes, all the dates and times will be automatically reformatted, and this PR is working great for us..